### DISCUSSION TITLE
Using a socket proxy instead of docker.sock

### DISCUSSION THREAD

**Discussion by itskagee** [almeidapaulopt/tsdproxy#130] [Q&A] [answered]
Hi,

Does tsdproxy support using a socket proxy like https://github.com/Tecnativa/docker-socket-proxy instead of a direct docker socket connection?

And if yes, then what API endpoints would I need to enable for it?

**Comment by almeidapaulopt** [comment, marked as answer]
Never used but it should work just fine.

API used, should be only these: 
/containers/json
/containers/*/json
/images/*/json
/events
/networks

**Reply by itskagee** [reply]
Got it.
Thanks!

How would I go about connecting tsdproxy to the socket proxy?
DOCKER_HOST env?

I didn't see any references in the docs.

**Reply by almeidapaulopt** [reply]
https://almeidapaulopt.github.io/tsdproxy/docs/serverconfig/

Docker section

**Reply by itskagee** [reply]
Whoops my bad.
No idea how I missed that.

Thank You!
